great_expectations/rule_based_profiler/data_assistant/volume_data_assistant.py

```diff
@@ -101,19 +134,24 @@ def _plot(
 
         batch_ids: KeysView[str]
         metric_values: MetricValues
-        batch_ids, metric_values = attributed_value.keys(), sum(
-            attributed_value.values(), []
-        )
+        batch_ids, metric_values = list(attributed_values_by_metric_name.values())[
+            0
+        ].keys(), sum(list(attributed_values_by_metric_name.values())[0].values(), [])
 
-        batch_numbers: List[int] = [batch + 1 for batch in range(len(batch_ids))]
+        idx: int
+        batch_numbers: List[int] = [idx + 1 for idx in range(len(batch_ids))]
 
         df: pd.DataFrame = pd.DataFrame(batch_numbers, columns=[x_axis])
         df["batch_id"] = batch_ids
-        df[metric] = metric_values
+        df[metric_name] = metric_values
```

Comment:
I like this change, but I also think it should be done on `x_axis` -> `x_axis_name`.

Reply:
A better name might actually be `domain_name` and `domain_type` instead of `x_axis`.

Reply:
@NathanFarmer Got it -- committed; thanks!